### DISCUSSION TITLE
How Can I ? (Window)

### DISCUSSION THREAD

**Discussion by DePhoegon** [otya128/winevdm#1212] [Q&A]
Is there anyway It would be possible to push a program into window mode & prevent it from freeking on a lose of focus?

**Comment by cracyc** [comment]
What program?

**Reply by DePhoegon** [reply]
The one I am having a 'time' with is 'Journeyman Project Turbo!'   The Dos/Win3.1/95 version.     

It's stable enough with the following
-no focus loss 
- not being forced to render at a resolution via windows. properties (it just takes up a fraction of the monitor & hangs as it thinks it lost focus)
- KB is used to progress the cutscenes, or they are allowed to play through entirely)
- Colors work as intended, despite it the whol 256 color warning (though that's ignorable or just a flag set on the exe to work past it)

I mean it runs perfectly fine otherwise.  Global Hotkeys works as is, The Mouse works, Video cutscenes work, fs access works. 
- Can be recorded with desktop recording.
-- I can't tell if the exe can be hooked properly, though I'm doubting that any program can.

I know it was never originally meant to be windowed ... like Ever on it's own.

**Reply by cracyc** [reply]
Since it was written to be be in fullscreen the developers likely never properly tested losing focus, trying to fix it would probably require patching the program since it's not a bug in winevdm.  If you want to make the window larger you can use something like integerscaler to magnify it or you can try it in scummvm which supports some versions of journeyman project.

**Reply by DePhoegon** [reply]
Would it be possible to introduce a window as if it was the desktop itself(that doesn't pass on if it'son focus or not), for an option on some runs?

If it becomes a problem I'll likely use a vm of sorts, though I'll also look into scummvm.  It's certainly seems easier then cracking win 3.1 into dosbox.

**Reply by cracyc** [reply]
I think that would be rather difficult.  There are tools to add a hotkey that locks the cursor to the focused window that you might try.
